2022-06-15 23:28:30,329 INFO    MainThread:14166 [wandb_setup.py:_flush():75] Loading settings from /home/aditta/.config/wandb/settings
2022-06-15 23:28:30,329 INFO    MainThread:14166 [wandb_setup.py:_flush():75] Loading settings from /media/aditta/UBUNTU/feedback-prize-effectiveness/code/wandb/settings
2022-06-15 23:28:30,329 INFO    MainThread:14166 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-06-15 23:28:30,329 INFO    MainThread:14166 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'trainer.py', 'program': 'trainer.py'}
2022-06-15 23:28:30,329 INFO    MainThread:14166 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-06-15 23:28:30,329 INFO    MainThread:14166 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-06-15 23:28:30,329 INFO    MainThread:14166 [wandb_init.py:_log_setup():405] Logging user logs to /media/aditta/UBUNTU/feedback-prize-effectiveness/code/wandb/run-20220615_232830-1z91s4d3/logs/debug.log
2022-06-15 23:28:30,329 INFO    MainThread:14166 [wandb_init.py:_log_setup():406] Logging internal logs to /media/aditta/UBUNTU/feedback-prize-effectiveness/code/wandb/run-20220615_232830-1z91s4d3/logs/debug-internal.log
2022-06-15 23:28:30,329 INFO    MainThread:14166 [wandb_init.py:init():439] calling init triggers
2022-06-15 23:28:30,329 INFO    MainThread:14166 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-06-15 23:28:30,329 INFO    MainThread:14166 [wandb_init.py:init():492] starting backend
2022-06-15 23:28:30,329 INFO    MainThread:14166 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-06-15 23:28:30,338 INFO    MainThread:14166 [backend.py:ensure_launched():219] starting backend process...
2022-06-15 23:28:30,345 INFO    MainThread:14166 [backend.py:ensure_launched():224] started backend process with pid: 14192
2022-06-15 23:28:30,347 INFO    MainThread:14166 [wandb_init.py:init():501] backend started and connected
2022-06-15 23:28:30,353 INFO    MainThread:14166 [wandb_init.py:init():565] updated telemetry
2022-06-15 23:28:30,355 INFO    MainThread:14166 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-06-15 23:28:47,249 INFO    MainThread:14166 [wandb_run.py:_on_init():1759] communicating current version
2022-06-15 23:28:51,380 INFO    MainThread:14166 [wandb_run.py:_on_init():1763] got version response upgrade_message: "wandb version 0.12.18 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-06-15 23:28:51,380 INFO    MainThread:14166 [wandb_init.py:init():625] starting run threads in backend
2022-06-15 23:28:51,808 INFO    MainThread:14166 [wandb_run.py:_console_start():1733] atexit reg
2022-06-15 23:28:51,809 INFO    MainThread:14166 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.REDIRECT
2022-06-15 23:28:51,809 INFO    MainThread:14166 [wandb_run.py:_redirect():1611] Redirecting console.
2022-06-15 23:28:51,811 INFO    MainThread:14166 [wandb_run.py:_redirect():1667] Redirects installed.
2022-06-15 23:28:51,811 INFO    MainThread:14166 [wandb_init.py:init():664] run started, returning control to user process
2022-06-15 23:28:51,909 INFO    MainThread:14166 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 1
2022-06-15 23:28:51,910 INFO    MainThread:14166 [wandb_run.py:_restore():1674] restore
2022-06-15 23:28:59,458 INFO    MainThread:14166 [wandb_run.py:_on_finish():1831] got exit ret: None
